sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample06_AlertConfigurationCrudOperations.cs

```diff
@@ -14,6 +13,7 @@ namespace Azure.AI.MetricsAdvisor.Samples
     public partial class MetricsAdvisorSamples : MetricsAdvisorTestEnvironment
     {
         [Test]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/22575")]
```

Comment:
we need to be careful next time you need to release as not having the samples running may cause problems that won't be detected (unless u run them manually).
This happened to us in DT and now we prioritize samples running than some tests

Reply:
Thank you for the advice. I'll mark the tracking issue as "blocking-release" just in case.